jenkins/bb-fuel.groovy

```diff
@@ -34,6 +34,7 @@ pipeline {
         string(name: 'BB_FUEL_VERSION', defaultValue: 'latest', description: '')
         booleanParam(name: 'PRERELEASE', defaultValue: false, description: 'Only applicable if BB_FUEL_VERSION = latest')
         string(name: 'ADDITIONAL_ARGUMENTS', defaultValue: '', description: 'Additional command line arguments')
+        booleanParam(name: 'MULTI_TENANCY_ENVIRONMENT', defaultValue: true, description: 'Enable multi tenancy')
```

Comment:
Default should be false. Otherwise it will be a breaking change

Reply:
done!